CHANGES.md

```diff
@@ -6,6 +6,18 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
+## Release 0.319.0
+
+* Improve the performance of get_view_names in SQLAlchemy. 
+([#267](https://github.com/trinodb/trino-python-client/issues/267)))
+* Fix value error with present but empty headers in the request.
+([#262](https://github.com/trinodb/trino-python-client/issues/262))
```

Comment:
please do not editorialise. And if you do please mention it so it can be reviewed. In this specific case the original entry from https://github.com/trinodb/trino-python-client/issues/260#issuecomment-1296849630 is more accurate and useful.

Reply:
Reminder.

Reply:
oops forgot to push this...done